modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/ClusterManager.java

```diff
@@ -347,4 +349,17 @@ public String getCoordinatorNodeAddress() {
         return addresses;
     }
 
+    /**
+     * Gets the broker's storage health status.
+     *
+     * @return true if healthy, else false.
+     */
+    public boolean getStoreHealth() {
+        boolean isMessageStoreOperational = AndesContext.getInstance().getMessageStore().isOperational("testString",
+                System.currentTimeMillis());
+        boolean isAndesContextStoreOperational = AndesContext.getInstance().getAndesContextStore().isOperational
+                ("testString", System.currentTimeMillis());
```

Comment:
I have a doubt whether getting "ClusterResourceHolder.getInstance()" would return null in a non-clustered deployment. Therefore I did not used it. 
Should I check that out and use it ?